wpiutil/src/main/native/include/wpi/circular_buffer.h

```diff
@@ -266,6 +266,13 @@ class circular_buffer {
     return m_data[(m_front + index) % m_data.size()];
   }
 
+  /**
+   * Returns a copy of the currently-stored data in the buffer.
+   *
+   * @return A copy of the underlying double array.
+   */
+  const std::vector<T> GetData() { return m_data; }
```

Comment:
This class was modeled after an STL container, and STL containers don't support things like this. You should be using circular_buffer's begin() and end() instead.

Reply:
ah; I'll fix that.